### PR TITLE
- Fix visualiser contentapiprefix

### PR DESCRIPTION
**Pull request by tomtollafield**

### CODE REVIEW

File: DFC.ServiceTaxonomy.Editor/Views/ContentVisualise.Button.cshtml [hunk 1]
```diff
@@ -22,7 +22,7 @@
             <a asp-action="@nameof(DFC.ServiceTaxonomy.GraphVisualiser.Controllers.VisualiseController.Viewer)"
                asp-controller="Visualise"
                asp-area="@typeof(DFC.ServiceTaxonomy.GraphVisualiser.Startup)!.Namespace"
-               asp-route-visualise="@graphSyncPartText"
+               asp-route-visualise="@(graphSyncPartText +"&graph=draft")"
```
Comment (tomtollafield) [inline DFC.ServiceTaxonomy.Editor/Views/ContentVisualise.Button.cshtml:25]: Is this the best way of doing this?

Comment (tomtollafield) [reply DFC.ServiceTaxonomy.Editor/Views/ContentVisualise.Button.cshtml:25]: (the graph name)

Comment (lazcool) [reply DFC.ServiceTaxonomy.Editor/Views/ContentVisualise.Button.cshtml:25]: I'd probably inject `GraphSyncHelper` and the 2 `ContentItemVersion`s into the view and call `GetIdPropertyValue()` to get `graphSyncPartText`.

**Comment by lazcool** [comment]
Obsoleted by later PR.

**Pull request closed without merging**